### DISCUSSION TITLE
Can't select a song unless I restart the game

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#23928] [Q&A]
### Type

Game doesn't function properly 

### Bug description

When I open **osu!lazer**, I can't use space/enter/mouse click to enter a beatmap,
restarting seems to fix that issue. 
happened to me once.

### Screenshots or videos

here's a link to that bug happening:
https://streamable.com/o3uask

### Version

osu! 2023.614.1-lazer

### Logs

[database.log](https://github.com/ppy/osu/files/11760082/database.log)
[input.log](https://github.com/ppy/osu/files/11760083/input.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/11760084/legacy-ipc.log)
[performance.log](https://github.com/ppy/osu/files/11760085/performance.log)
[runtime.log](https://github.com/ppy/osu/files/11760086/runtime.log)
[updater.log](https://github.com/ppy/osu/files/11760087/updater.log)

**Comment by peppy** [comment]
Did you figure this out?
